mozilla_aws_cli/login.py

```diff
@@ -260,6 +233,7 @@ def login(self):
             # start up the listener, figuring out which port it ran on
             logger.debug("About to start listener running on port {}".format(port))
             listen(self)
+        return True
```

Comment:
I think most of our code has a space before the return statement at the end of the function.

Reply:
I'll take care of this in #183